integration_tests/elastic_search_test.go

```diff
@@ -0,0 +1,60 @@
+package integration_test
+
+import (
+	. "github.com/onsi/ginkgo"
+
+	"github.com/alphagov/paas-compose-broker/integration_tests/helper"
+	"os"
+)
+
+var _ = Describe("Broker Compose Integration", func() {
+
+	Context("ElasticSearch", func() {
+
+		BeforeEach(func() {
+			if os.Getenv("SKIP_COMPOSE_API_TESTS") == "true" {
```

Comment:
OK so I've moved the actual env check to the test entrypoint to make it bit more DRY, but didn't want to change the logic of using Skip in the BeforeEach since I thought it was important to keep the test output of all the skips.